### PR TITLE
GH-550 Add missing tests for Panda Utilities

### PR DESCRIPTION
**Pull request by wamukat**
This PR may increase test coverage a little for Panda Utilities.
- org.panda_lang.utilities.commons.console.Colored
- org.panda_lang.utilities.commons.ValidationUtils

related to issue #550.

### CODE REVIEW

File: panda-utilities/src/test/java/org/panda_lang/utilities/commons/ValidationUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    void notNull__null_argument_throws_exception() {
+        Assertions.assertThrows(
+                IllegalArgumentException.class,
+                () -> ValidationUtils.notNull(null)
+        );
+    }
+
+    @Test
+    void notNull__non_null_argument_returns_argument() {
+        final String value = "testValue";
```
Comment (dzikoysk) [inline panda-utilities/src/test/java/org/panda_lang/utilities/commons/ValidationUtilsTest.java:17]: We don't really use final notation for local variables. I also think that this case might be shortened by moving value and actual value directly to assertEquals statement - exactly as in the method above

Comment (dzikoysk) [inline panda-utilities/src/test/java/org/panda_lang/utilities/commons/ValidationUtilsTest.java]: We don't really use final notation for local variables. I also think that this case might be shortened by moving value and actual value directly to assertEquals statement - exactly as in the method above

**Review by dzikoysk — CHANGES_REQUESTED** [review]

**Comment by dzikoysk** [comment]
Thanks for PR! ❤️ I've made a review and I have just one note as you see, after that I'll be able to merge it :)

File: panda-utilities/src/test/java/org/panda_lang/utilities/commons/ValidationUtilsTest.java (hunk 1 above)
Comment (wamukat) [reply panda-utilities/src/test/java/org/panda_lang/utilities/commons/ValidationUtilsTest.java]: @dzikoysk Thanks for your review!
Please check the commit a73edd4 to make sure it's fixed.

**Review by dzikoysk — APPROVED** [review]

**Pull request merged**